schemas/fingerprint-server-api.yaml

```diff
@@ -1768,3 +1834,67 @@ components:
       required:
         - code
         - message
+    EventUpdateRequest:
+      type: object
+      properties:
+        linkedId:
+          type: string
+          description: LinkedID value to assign to the existing event
+        tags:
```

Comment:
@petrpalata why is it `tags` when the JS agent property is `tag`? 
Can we unify it? 

https://dev.fingerprint.com/docs/js-agent#tag

Reply:
Seems to be a miss in the spec, I thought it's called `tags` in the Server API response. We need to fix this in the response itself but can update it here in the meantime.